src/app/[lang]/locales/en/admin.json

```diff
@@ -1,6 +1,21 @@
 {
+  "DashboardTitle": "Admin dashboard",
   "CreateTag": {
     "addNewTag": "Add new tag",
-    "existingTags": "Existing tags:"
+    "tagsLabel": "Tags"
+  },
+  "EditUsers": {
+    "label": "Users",
+    "infoText": "Sign in again to apply changes to user roles",
```

Comment:
```suggestion
    "infoText": "Changes in user roles are applied when user signs in again",
```

Reply:
I thought this might be a tad bit clearer (the admin does not have to sign in again for changes in other users' user roles to be applied)